Apps/PackageTest/0.65.0/package.json

```diff
@@ -11,7 +11,7 @@
     "windows": "react-native run-windows"
   },
   "dependencies": {
-    "@babylonjs/core": "^5.2.0",
```

Comment:
Looks like the package-lock.json changes associated with this change are missing.